src/styles/HeaderBar.scss

```diff
@@ -4,6 +4,7 @@
 
 .catalog-top-app-bar {
   @include mdc-top-app-bar-fill-color($material-color-grey-900);
+  z-index: 2;
```

Comment:
Is this still necessary with the padding-top and margin adjusted?

Reply:
Yes, the buttons end up above the top app bar because of stacking context.